flocker/dockerplugin/test/test_api.py

```diff
@@ -64,15 +66,17 @@ def test_unmount(self):
         '1.9.0',
         'This test uses the v2 plugin API, which requires Docker >=1.9.0'
     )
-    def test_create_with_opts(self):
+    def test_create_with_profile(self):
         """
         Calling the ``/VolumerDriver.Create`` API with an ``Opts`` value
-        in the request body JSON ignores this parameter and creates
-        a volume with the given name.
+        of "profile=[gold,silver,bronze] in the request body JSON create a
+        volume with a given name with [gold,silver,bronze] profile.
         """
+        default_profiles = ["gold", "silver", "bronze"]
```

Comment:
This would probably be better using `hypothesis`. Something like
```python
@given(profile = sampled_from(["gold", "silver", "bronze"])
def test_...(self, profile):
    ...
```